### ISSUE TITLE
Reduce redundant database operations

### ISSUE THREAD

**Issue by ghost** [zachtheclimber/mpv#28]
Right now, whenever a someone uses the app, their entire tick list is imported into the database. If you are a return user, the app drops your current table, then builds a new one. In the interests of reducing the wait time for users with large tick lists (our most consistent users, I'd guess), I think running a check for each tick, and only inserting new ones, might prove to be faster, at least marginally. I'm going to create a branch to work on this and see.

The only major side effect I can imagine is if you are an existing MPV user, and you delete a tick on Mountain Project, there is no way for the app to know, and therefore it will still be counted in your MPV stats. I think this is a minor trade off that will likely go unnoticed.

**Comment by ghost** [comment]
Fixed with PR #29 .

**Comment by ghost** [comment]
A few metrics I forgot to post previously:

Before turning autocommit off:
```
fetch_user took 0.5528278351 seconds
parse_user_data took 0.0010428429 seconds
fetch_tick_list took 6.1188955307 seconds
parse_tick_list took 0.0302412510 seconds
db_connect took 0.0143609047 seconds
get_pairs took 0.0041635036 seconds
db_close took 0.0017292500 seconds
db_load took 6.2992408276 seconds
db_connect took 0.0268645287 seconds
db_close took 0.0005669594 seconds
```

After turning autocommit off:
```
fetch_user took 0.5787246227 seconds
parse_user_data took 0.0013735294 seconds
fetch_tick_list took 6.3166189194 seconds
parse_tick_list took 0.0127792358 seconds
db_connect took 0.0089380741 seconds
get_pairs took 0.0029284954 seconds
db_close took 0.0031485558 seconds
db_load took 0.6354691982 seconds
db_connect took 0.0120861530 seconds
db_close took 0.0006134510 seconds
```
